index.js

```diff
@@ -147,37 +183,17 @@ function stopSauceConnect (done) {
 	{
 		keystone: <keystone instance>,		// REQUIRED
 		runSelenium: [ true | false ]			// DEFAULTS TO FALSE
-		sauceConnect: [ true | false ]		// DEFAULTS TO TRUE
 	}
 */
 function start (options, callback) {
```

Comment:
This start function doesn't do much now, could the two `async.series` bits in this file be merged into a single one to make it clearer?